source/application/REIXS/REIXSAppController.cpp

```diff
@@ -228,6 +227,7 @@ void REIXSAppController::shutdown() {
 
 	// Make sure we release/clean-up the beamline interface
 	AMBeamline::releaseBl();
+	AMStorageRing::releaseStorageRing();
```

Comment:
When I moved that call up the hierarchy in #813 I removed it from here at the same time